src/openapi/spec/register-client-schema.ts

```diff
@@ -0,0 +1,39 @@
+import { createSchemaObject, CreateSchemaType } from '../openapi-types';
+
+export const schema = {
+    type: 'object',
+    required: ['appName', 'interval', 'started', 'strategies'],
+    properties: {
+        appName: {
+            type: 'string',
+        },
```

Comment:
Nice! I'd love to see `description`s of the schema and each individual property too, so that it's extra obvious to users what the object is used for.

Reply:
Sure, but we should start with the official one (and maybe a way to share these?)
https://github.com/Unleash/unleash/blob/main/src/lib/openapi/spec/client-application-schema.ts

Reply:
Oh, wait, is this a duplicate of something that exists in Unleash proper?

Reply:
And regardless: if we're duplicating it, we should add description to both models. Doesn't really matter which one we do first 💁🏼

Reply:
agreed. I will update this spec first and propagate it to the unleash project. Later we should find ways to share these, I suspect this will need to come as a side-effect of the embedded proxy feature anyways.